server/app/views/admin/programs/predicates/PredicateValuesInputFragment.html

```diff
@@ -101,6 +106,39 @@
   </div>
 </th:block>
 
+<!-- Date question type -->
+<th:block
+  th:fragment="dateValuesInput(valueOptions, valueInputId, secondValueInputId, secondValueInputGroupId)"
+>
+  <div
+    class="usa-hint"
+    th:id="${valueInputHintId}"
+    th:text="#{content.multipleValuesInputHint}"
+  ></div>
+  <input
+    class="usa-input"
+    inputmode="numeric"
+    th:id="${valueInputId}"
+    th:name="${valueInputId}"
+    aria-required="true"
+    aria-invalid="false"
+    date-value
+  />
+
+  <div th:id="${secondValueInputGroupId}" class="margin-top-1">
+    <span class="margin-left-1" th:text="#{content.and}"></span>
+    <input
+      class="usa-input"
+      inputmode="numeric"
+      th:id="${secondValueInputId}"
+      th:name="${secondValueInputId}"
+      aria-required="true"
+      aria-invalid="false"
+      date-value
```

Comment:
```suggestion
    data-date-value
```

(blocking) Custom attributes need to be prefixed with `data-`